datadog-ci.gemspec

```diff
@@ -4,35 +4,39 @@ require_relative "lib/datadog/ci/version"
 
 Gem::Specification.new do |spec|
   spec.name = "datadog-ci"
-  spec.version = Datadog::CI::VERSION
+  spec.version = Datadog::CI::VERSION::STRING
+  spec.required_ruby_version = [
+    ">= #{Datadog::CI::VERSION::MINIMUM_RUBY_VERSION}",
+    "< #{Datadog::CI::VERSION::MAXIMUM_RUBY_VERSION}"
+  ]
+  spec.required_rubygems_version = ">= 2.0.0"
   spec.authors = ["Datadog, Inc."]
   spec.email = ["dev@datadoghq.com"]
 
   spec.summary = "Datadog CI visibility for your ruby application"
-  # spec.description = "TODO: Write a longer description or delete this line."
-  # spec.homepage = "TODO: Put your gem's website or public repo URL here."
-  spec.license = "BSD-3-Clause"
-  # spec.required_ruby_version = ">= 2.6.0"
+  spec.description = <<-DESC
+  datadog-ci is a Datadog's CI visibility library for Ruby. It traces
+  tests as they are being executed and brings developers visibility into
+  their CI pipelines.
+  DESC
 
-  spec.metadata["allowed_push_host"] = "TODO: Set to your gem server 'https://example.com'"
+  spec.homepage = "https://github.com/DataDog/datadog-ci-rb"
+  spec.license = "BSD-3-Clause"
 
-  # spec.metadata["homepage_uri"] = spec.homepage
-  # spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
-  # spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
+  spec.metadata["allowed_push_host"] = "https://rubygems.org"
+  spec.metadata["changelog_uri"] = "https://github.com/DataDog/datadog-ci-rb/blob/main/CHANGELOG.md"
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = "https://github.com/DataDog/datadog-ci-rb"
+
+  spec.files =
```

Comment:
Can add a test similar to https://github.com/DataDog/dd-trace-rb/blob/master/spec/ddtrace/release_gem_spec.rb ?

Reply:
yes, will do